.dotnet/src/Custom/Files/FileClient.Protocol.cs

```diff
@@ -209,4 +210,52 @@ public virtual ClientResult DownloadFile(string fileId, RequestOptions options)
         using PipelineMessage message = CreateDownloadFileRequest(fileId, options);
         return ClientResult.FromResponse(_pipeline.ProcessMessage(message, options));
     }
+
+    [Experimental("OPENAI001")]
+    public virtual async Task<ClientResult> CreateUploadAsync(BinaryContent content, RequestOptions options = null)
```

Comment:
Could you add some simple doc comments for these methods?

Reply:
Done!